thunder/torch/__init__.py

```diff
@@ -2912,7 +2912,7 @@ def scatter_add_(a: TensorLike, /, dim: int, index: TensorLike, src: TensorLike)
     return prims.copy_(scatter_add(a, dim, index, src), a)
 
 
-@torchsymbol(torch.take_along_dim)
+@torchsymbol(torch.take_along_dim, torch.Tensor.take_along_dim, is_method=True)
```

Comment:
Question (for my understanding): I thought `is_method=True` would be sufficient here due to this logic: https://github.com/Lightning-AI/lightning-thunder/blob/59467aa5e401ea1c4d2c4e6f9e94d1dbe147a19e/thunder/torch/__init__.py#L183-L188 and I'm speculating we register `torch.Tensor.take_along_dim` to `_torch_to_thunder_function_map` twice, which I wouldn't find harmful but I'm just curious

Reply:
Thank you I missed this, let me remove the duplicates